src/components/EmptyContent.vue

```diff
@@ -1,140 +0,0 @@
-<!--
- - @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
- -
- - @author John Molakvoæ <skjnldsv@protonmail.com>
- -
- - @license AGPL-3.0-or-later
- -
- - This program is free software: you can redistribute it and/or modify
- - it under the terms of the GNU Affero General Public License as
- - published by the Free Software Foundation, either version 3 of the
- - License, or (at your option) any later version.
- -
- - This program is distributed in the hope that it will be useful,
- - but WITHOUT ANY WARRANTY; without even the implied warranty of
- - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- - GNU Affero General Public License for more details.
- -
- - You should have received a copy of the GNU Affero General Public License
- - along with this program. If not, see <http://www.gnu.org/licenses/>.
- -
- -->
-
-<template>
-	<div v-if="isReady" class="emptycontent">
-		<!-- eslint-disable-next-line vue/no-v-html (because it's an SVG file) -->
-		<div v-if="haveIllustration" class="illustration" v-html="illustration" />
```

Comment:
@artonge regression, we were not using the vue component.
So you, migrating to it, broke support for illustrations as well as default empty contents :)
https://github.com/nextcloud/photos/blob/ce56825aa759b6ba8d5175579af75818cceeea11/src/views/Tags.vue#L27-L29
https://github.com/nextcloud/photos/blob/ce56825aa759b6ba8d5175579af75818cceeea11/src/views/Tags.vue#L31-L36

Reply:
Seen, and on it ;) trying to abstract those views to have more consistency between them